docs/ingesting_data/Ingesting_data_with_panpipes.md

```diff
@@ -135,44 +119,33 @@ scr:
 
 ```
 normalisation_methods: clr
-# CLR parameters:
-# margin determines whether you normalise per cell (as you would RNA norm), 
-# or by feature (recommended, due to the variable nature of adts). 
+
+#-----------------------------------------------
+# Centered log ratio (CLR) normalization options
+
+# margin determines whether you normalise per cell (as you would for RNA),
+# or by feature (recommended, due to the variable nature of prot assays).
 # CLR margin 0 is recommended for informative qc plots in this pipeline
-# 0 = normalise colwise (per feature)
-# 1 = normalise rowwise (per cell)
+# 0 = normalise rowwise (per feature)
+# 1 = normalise colwise (per cell)
```

Comment:
I just noticed that there seems to be a difference between the `pipeline.yml` itself and the part shown in the tutorial. @bio-la, could you please double-check that it is correct this way? In the typical adata, cells would be stored row-wise, no?

Reply:
margin 0 is rows, which in anndata is cells
margin 1 is cols, which in anndata is features

# 0 = normalise rowwise (per cell)
# 1 = normalise colwise (per feature)

we recommend the default should be per feature which is dissimilar to seurat's

and please [fix the documentation](https://panpipes-pipelines.readthedocs.io/en/latest/usage/normalization_methods.html)

and in the preprocess yaml documentation check the statement: 



`margin determines whether you normalise per cell (as you would RNA norm), 
or by feature (recommended, due to the variable nature of adts). `